client/src/components/RequestModal.js

```diff
@@ -171,15 +176,24 @@ export default class RequestModal extends Component {
             >
                 <Modal.Header>Schedule a meeting with {this.state.alumni.name}!</Modal.Header>
                 <Modal.Content>
-                    <Grid>
+                    <Grid stackable>
                         <Grid.Row columns={"equal"}>
-                            <Grid.Column width={3}>
+                            <Grid.Column width={4}>
                                 <Image
                                     floated='left'
                                     size='small'
                                     src={this.state.alumni.imageURL}
                                     rounded
                                 />
+                                <TimeZoneDropdown
+                                    userDetails={this.props.userDetails}
+                                    role={this.props.role}
+                                    liftTimezone={this.handleOffsetChange}
+                                    compact={true}
+                                />
+                                <Segment compact inverted color='blue tertiary'>
```

Comment:
Made a warning here because it doesn't feel like expected behavior - one option is to refresh at toggleModal in alumniDirectory which I think might be a better option